api/client/service/update_test.go

```diff
@@ -68,8 +68,13 @@ func TestUpdateEnvironment(t *testing.T) {
 
 	updateEnvironment(flags, &envs)
 	assert.Equal(t, len(envs), 2)
-	assert.Equal(t, envs[0], "tokeep=value")
-	assert.Equal(t, envs[1], "toadd=newenv")
+	// Order has been removed in updateEnvironment (map)
+	if envs[0] == "tokeep=value" {
+		assert.Equal(t, envs[1], "toadd=newenv")
+	} else {
+		assert.Equal(t, envs[0], "toadd=newenv")
+		assert.Equal(t, envs[1], "tokeep=value")
+	}
```

Comment:
How about sorting the slices instead of this condition? The `if` makes the test a bit harder to read.